packages/react/src/components/CardEditor/CardEditForm/CardEditFormItems/DataSeriesFormItems/DataSeriesFormSettings.jsx

```diff
@@ -30,6 +30,8 @@ const propTypes = {
       includeZeroOnYaxis: PropTypes.bool,
       timeDataSourceId: PropTypes.string,
       showLegend: PropTypes.bool,
+      /** maximum amounts of data points to render in the graph */
+      maximumDataPoints: PropTypes.bool,
```

Comment:
bool?

Reply:
nope, thanks for catching.  The data point limitation doesn't actually affect the cards themselves currently, only is stored in the JSON and up to the downstream consumers to use when they fetch their data.  Should I add to the chart itself?

Reply:
Got it. No, I don't think it's necessary to add to the chart itself, but maybe clarify that in the prop-type comment?